.circleci/config.yml

```diff
@@ -38,7 +38,12 @@ do_steps: &do_steps
       name: build DPDK
       command: |
         if [ "$MKARG" = "dpdk=yes" ]; then
-          sudo apt-get update && sudo apt-get install -y linux-headers-$(uname -r)
+          sudo apt-get update
+          if ! sudo apt-get install -y linux-headers-$(uname -r) ; then
+             sudo mkdir -p /lib/modules/ && cd /lib/modules && \
```

Comment:
Why run mkdir /lib/modules? If /lib/modules is not there the link below will point to something nonexistent?

Reply:
you're right. it doesn't need.
deleted.